projects_integration_test.go

```diff
@@ -123,7 +123,7 @@ func TestProjectsCreate(t *testing.T) {
 			Name: badIdentifier,
 		})
 		assert.Nil(t, w)
-		assert.EqualError(t, err, ErrInvalidName.Error())
+		assert.Contains(t, err.Error(), "invalid attribute\n\nName may only contain")
```

Comment:
Is this sentence `Name may only contain` complete? or is it missing part of it, like `Name may only contain alphanumeric characters and underscore`?

Reply:
The error comes from the API, but I don't want to fully validate the response in case it ever evolves.